modules/ROOT/pages/tools/io-tool.adoc

```diff
@@ -0,0 +1,110 @@
+= IO-Tool
+:imagesdir: 
+
+== What is the IO-tool?
+
+https://io.my-agrirouter.com/[The agrirouter InOut-Tool] is a tool to simulate a remote endpoint and to learn to communicate via agrirouter. 
+
+image::io-tool/settings.png[]
+
+
+Using the IO-Tool you can 
+
+* exchange data
+* send and receive telemetry data
+* set preferences for the data exchange
+* check the logs and debug
+* simulate end points and set routings
+
+
+== Getting Started
+
+First you need to log in to your account, which can be requested from Agrirouter support (support@my-agrirouter.com).
```

Comment:
This is a bit "Upside down": "First you need to log in, oh wait, you need an account oh wait you have to register":

Better do this in order
=== Requesting an Account ("The tool has no self service registration, please send an email like this to ...."
=== Log In  

This way the reader has a straight forward process and gets answered his question of "Where to get an account" directly in the header.